src/components/assignees/Assignees.jsx

```diff
@@ -0,0 +1,81 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+import { Avatar, Popover } from 'antd'
+import './style.css'
+
+
+class Assignees extends Component {
+
+  getAssignees() {
+    const members = this.props.assignees.sort((a, b) => a.initials > b.initials).slice(0)
+    if (this.props.cardResponsible !== null) {
+      // if responsible is in members, sets responsible to be the first user
+      if (members.find(element => element.id === this.props.cardResponsible.id) === true) {
```

Comment:
Just a tip: you don't need to append `=== true`, it's already a boolean assertion